pg_wait_sampling.c

```diff
@@ -39,36 +34,42 @@
 
 PG_MODULE_MAGIC;
 
-void		_PG_init(void);
-
+/* Marker whether extension is setup in shared mode */
 static bool shmem_initialized = false;
 
+/* Global settings */
+int MaxProfileEntries = 5000;
+int HistoryBufferSize = 5000;
+int HistoryPeriod = 0;
+int ProfilePeriod = 10;
+bool WhetherProfilePid = true;
+bool WhetherProfileQueryId = true;
```

Comment:
I think we need to prefix public globals to avoid name clashes.

Reply:
Makes sense. Will be done.